common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -104,4 +104,13 @@ trait ABTestSwitches {
     sellByDate = Some(LocalDate.of(2023, 2, 1)),
     exposeClientSide = true,
   )
+  Switch(
+    ABTests,
+    "ab-sign-in-gate-copy-test",
+    "Test the impact of changing the copy in the sign in gate",
+    owners = Seq(Owner.withEmail("personalisation.dev@theguardian.com")),
```

Comment:
While this email address would make logical sense it doesn't exist. There is only `personalisation@guardian.co.uk`

Reply:
Updated the email address